controllers/hosting/hostingdeployment_controller_test.go

```diff
@@ -40,6 +40,7 @@ import (
 	commonv1 "github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/common"
 	endpointconfigv1 "github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/endpointconfig"
 	hostingv1 "github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/hostingdeployment"
+	. "github.com/aws/amazon-sagemaker-operator-for-k8s/controllers"
```

Comment:
Please avoid using dot imports - they give warnings in golint. I've been trying to remove our use of these in the refactor work.